src/components/Sprints/SelectedSprintComonent.js

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useReducer } from 'react';
+import {
+  Select,
+  Container,
+  Typography,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
+} from '@material-ui/core';
+import RemoveIcon from '@material-ui/icons/Remove';
+import { makeStyles } from '@material-ui/core/styles';
+import '../../App.css';
+import * as dbUtils from '../../dbUtils';
+
+const useStyles = makeStyles({
+  formControl: {
+    minWidth: '100%',
+  },
+  inputLabel: {
+    color: '#bbb',
+  },
+  userInput: {
+    color: 'white',
+    width: '100%',
+  },
+});
+
+const SprintSelectionComponent = () => {
+  const classes = useStyles();
+
+  const initialState = {
+    sprint: [],
+    project: {},
+    MenuSelection: '',
+  };
+  const [state, setState] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    initialState
+  );
+
+  useEffect(() => {
+    loadSessionStorage();
+  }, []);
+
+  const loadSessionStorage = () => {
+    setState({
+      sprint: JSON.parse(sessionStorage.getItem('sprint')),
+      project: JSON.parse(sessionStorage.getItem('project')),
+    });
+  };
+
+  const handleSelection = async (e) => {
+    //update the local project to not contain the user story
+    let updatedSprint = state.sprint;
+
+    updatedSprint.userStories = [...updatedSprint.userStories, e.target.value];
+
+    let updatedProject = state.project;
+    updatedProject.productBacklog = updatedProject.productBacklog.filter(
+      (item) => item !== e.target.value
+    );
+
+    setState({
+      project: updatedProject,
+      sprint: updatedSprint,
+    });
+
+    await updateStorage(updatedProject, updatedSprint);
+  };
+
+  const removeTask= async(story) =>{
+
+    //Update local storage
+    let updatedSprint = state.sprint;
+    let updatedProject = state.project;
+
+    updatedSprint.userStories = updatedSprint.userStories.filter((item)=>item!==story);
+    updatedProject.productBacklog = [...updatedProject.productBacklog, story];
+
+    setState({
+      project: updatedProject,
+      sprint: updatedSprint,
+    });
+
+    await updateStorage(updatedProject, updatedSprint);
+    
+  }
+
+  const updateStorage = async (updatedProject, updatedSprint)=>{
+    sessionStorage.setItem('sprint', JSON.stringify(updatedSprint));
+    sessionStorage.setItem('project', JSON.stringify(updatedProject));
+
+    //Add it to the sprint and update
+    await dbUtils.updateProject(updatedProject);
+    await dbUtils.updateSprint(updatedSprint);
+
```

Comment:
here's a case for making the sprints a sub-collection instead of a separate one, this could be done in one function call that would ensure consistency in the data